### DISCUSSION TITLE
py mammotion in other platforms?

### DISCUSSION THREAD

**Discussion by nieand1985** [mikey0000/Mammotion-HA#176] [Ideas]
Hello everyone,
is it possible to get the pymammotion integration on other platforms, such as io broker, Mqtt, Loxberry or Docker?

I am unfortunately an absolute beginner in this area and would appreciate any help. 

Many thanks in advance

**Comment by mikey0000** [comment]
Can you open this on pymammotion repo please

**Comment by mikey0000** [comment]
> Hello everyone, is it possible to get the pymammotion integration on other platforms, such as io broker, Mqtt, Loxberry or Docker?
> 
> I am unfortunately an absolute beginner in this area and would appreciate any help.
> 
> Many thanks in advance

In short if you are a complete beginner, look at MQTT bridges but you may be trying to do something a bit out of your league and I mean this politely.
